eventsourcing/src/main/scala/com/evolutiongaming/akkaeffect/eventsourcing/Engine.scala

```diff
@@ -266,6 +265,80 @@ object Engine {
     } yield engine
   }
 
+  /**
+   * Cats-effect based implementation, expected to be used '''only in tests'''
+   */
+  def of[F[_]: Async, S, E](
```

Comment:
does it work anyhow differently than the original one?

Reply:
I would say no, but I have not tested it properly. Main difference is that no akka required

Reply:
will it be hard - using existing unit test for both cases: akka based and this impls?

Reply:
I've improved the implementation to be inline with already existing Akka based impl